orchagent/bfdorch.cpp

```diff
@@ -81,6 +100,21 @@ BfdOrch::~BfdOrch(void)
     SWSS_LOG_ENTER();
 }
 
+std::string BfdOrch::replaceFirstTwoColons(const std::string &input) {
```

Comment:
can you please rename to something like createStateDBKey.

Reply:
Done